### PR TITLE
Update readme #3045

### PR DESCRIPTION
**Pull request by alhambrav**
### Ticket reference or full description of what's in the PR
Update readme #3045

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Crafter CMS
 
-Crafter CMS is an open source content management system for Web sites, mobile apps, VR and more. You can learn more about Crafter here: http://docs.craftercms.org/en/3.1/index.html
+Crafter CMS is an open source content management platform for building digital experience applications (React, Vue, Angular, HTML, AR/VR, Mobile, Headless, …).  You can learn more about Crafter here: http://docs.craftercms.org/en/3.1/index.html
```
Comment (sumerjabri) [inline README.md]: Why not use the same initial blurb we now have in 3.1 docs? (not the whole thing, just the first section).

File: README.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Replace {BRANCH} or {TAG NAME} or \<branch\> with the branch and tag you'd like
 When using a tag-based build, you're essentially cloning a point in time to build that specific version of Crafter CMS. That implies that you won't be able to update/nor push changes back.
 
 # 3. Build a Developer's Environment
-Crafter CMS is built along a microservices architecture, and as such, comprises a number of head-less, RESTful, modules that work together to provide the final solution. In this section, we'll start with the simple case of _build everything_/_run everything_, and then move on to building/hacking individual modules.
+Crafter CMS is built along a microservices based, serverless (Docker + Kubernetes), architecture, and as such, comprises a number of headless API-first (GraphQL, REST, in-process)  modules that work together to provide the final solution. In this section, we'll start with the simple case of _build everything_/_run everything_, and then move on to building/hacking individual modules.
```
Comment (sumerjabri) [inline README.md]: microservices-based

Comment (sumerjabri) [reply README.md]: We must also indicate that you don't have to run Kuber to get a local env.

A basic dev env, like this section, is probably not going to be based on a Kuber cluster. Let's discuss.

Comment (avasquez614) [reply README.md]: Yeah I don't think we need to reference Kuber or Docker hear except to say that you can build the Docker images.

**Pull request merged**